pyqtgraph/parametertree/ParameterTree.py

```diff
@@ -130,6 +134,16 @@ def contextMenuEvent(self, ev):
         if hasattr(item, 'contextMenuEvent'):
             item.contextMenuEvent(ev)
             
+    def updatePalette(self):
+        """
+        - called when application palette changes -
```

Comment:
Pretty sure that this is what makes sphinx confused, and that
"Called when the application palette changes"
would not cause any trouble.